platform/persistence/platform-persistence-core-api/src/main/java/org/codice/ddf/persistence/PersistentStore.java

```diff
@@ -51,6 +52,15 @@
     public void add(String type, Map<String, Object> properties) throws PersistenceException;
 
     /**
+     * Adds a collection of item of specified type.
+     *
+     * @param type
```

Comment:
✏️  Since this is an API, we should provide descriptions for the different parameters and exception. Especially important for the `Collection<Map<>>` parameter where we probably should say what things are valid.